.github/workflows/sql-pr.yml

```diff
@@ -0,0 +1,29 @@
+name: PR SQL Benchmarks
+
+on:
+  pull_request:
+    types: [ labeled, synchronize ]
+    branches: [ "develop" ]
+  workflow_dispatch: { }
+
+permissions:
+  actions: write
+  contents: read
+  pull-requests: write
+  id-token: write
+
+jobs:
+  label_trigger:
+    runs-on: ubuntu-latest
+    if: ${{ contains(github.event.head_commit.message, '[benchmark-sql]') || github.event.label.name == 'benchmark-sql' && github.event_name == 'pull_request' }}
```

Comment:
Is the idea that we can run the SQL benchmarks without running random_access etc.?

Reply:
yes